prow/repoowners/repoowners.go

```diff
@@ -270,6 +280,11 @@ func (o *RepoOwners) walkFunc(path string, info os.FileInfo, err error) error {
 		return nil
 	}
 	filename := filepath.Base(path)
+
+	dirBlacklist := sets.NewString(o.dirBlacklist...)
+	if len(dirBlacklist) == 0 {
+		dirBlacklist = defaultDirBlacklist
```

Comment:
Feels like we should separate the generation of the list (getter func) and the use of it, this blurs that line